modules/aws-emr-cluster/main.tf

```diff
@@ -60,6 +60,9 @@ resource "aws_emr_cluster" "emr-cluster" {
     }
   }
 
+  custom_ami_id      = var.emr_custom_ami_id
+  custom_ami_version = var.emr_custom_ami_version
```

Comment:
I'm only looking for the documentation on the latest version so maybe I'm missing something but is `custom_ami_version` an argument you can set on the aws_emr_cluster resource? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/emr_cluster